src/event-manager.js

```diff
@@ -173,8 +178,13 @@ class DefaultEventStrategy {
       delegatedHandlers = this.delegatedHandlers;
       handlerEntry = delegatedHandlers[targetEvent] || (delegatedHandlers[targetEvent] = new DelegateHandlerEntry(targetEvent));
       let delegatedCallbacks = target.delegatedCallbacks || (target.delegatedCallbacks = {});
+      if (delegatedCallbacks[targetEvent]) {
+        emLogger.error('already have a callback for event', {event: targetEvent, callback: callback});
```

Comment:
Maybe add some note here that logging is only for debugging purpose, log content can be changed at any time so there won't be any accidental breakage